examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkLongImReadCommand.kt

```diff
@@ -36,7 +36,30 @@ class PairOnNetworkLongImReadCommand(
     }
 
     override fun onReport(nodeState: NodeState) {
-      logger.log(Level.INFO, "Read receve onReport")
+      logger.log(Level.INFO, nodeState.toString())
+
+      val attribute =
+        nodeState
+          .getEndpointState(0)
+          ?.getClusterState(CLUSTER_ID_BASIC)
+          ?.getAttributeState(ATTR_ID_LOCAL_CONFIG_DISABLED)
+
+      if (!(attribute != null && attribute.getJson().toString().equals("""{"16:BOOL":false}"""))) {
+        setFailure("read attribute failure")
+        return
+      }
+      val event =
+        nodeState
+          .getEndpointState(0)
+          ?.getClusterState(CLUSTER_ID_BASIC)
+          ?.getEventState(EVENT_ID_START_UP)
+      if (
+        !(event != null && event[0].getJson().toString().equals("""{"0:STRUCT":{"0:UINT":1}}"""))
```

Comment:
Could we also add test for TLV payload, that is what we actually used right now

Reply:
done, thanks